### PR TITLE
Added an extra optionale step to the installation process

### PR DESCRIPTION
**Pull request by JeffreyPerplex**
When having a workshop yesterday at UmbracoDK Festival I could not install Umbraco v9 although I followed all the steps. Magician Sebastiaan helped me out and I wanted to make sure that his magic solution was included in the documentation as well

### CODE REVIEW

**Comment by busrasengul** [comment]
Thank you @JeffreyPerplex 
This is great!! We'll review this soon :)

File: UmbracoNetCoreUpdates.md [hunk 1]
```diff
@@ -137,6 +137,16 @@ You will now have a new project with the name `MyCustomUmbracoProject`, or which
 
 The new project can be opened and run using your favorite IDE or you can continue to use the CLI commands.
 
+### [Optional] Error message 'No template found matching: Umbraco'
+If you are sure that you've installed the templates (see the previous step) but the command prompt keeps telling you that there is "No template found matching: Umbraco" it can help to specificy the specific version and nuget source.
```
Comment (marcemarc) [inline UmbracoNetCoreUpdates.md:141]:
```suggestion
If you are sure that you've installed the templates (see the previous step) but the command prompt keeps telling you that there is "No template found matching: Umbraco" then you can 'work around' by detailing the specific version in the `--nuget-source` option.
```

File: UmbracoNetCoreUpdates.md [hunk 2]
```diff
@@ -137,6 +137,16 @@ You will now have a new project with the name `MyCustomUmbracoProject`, or which
...
+
+![foutmelding opdrachtprompt](https://user-images.githubusercontent.com/12457148/141447451-538b6eca-3c99-4e69-82d9-d58e8913e951.png)
+
+1. Use a command prompt of your choice to specify the specific sources. Please note that you can update the version (9.0.1.) to the latest version that is available.
```
Comment (marcemarc) [inline UmbracoNetCoreUpdates.md:145]:
```suggestion
```

File: UmbracoNetCoreUpdates.md [hunk 3]
```diff
@@ -137,6 +137,16 @@ You will now have a new project with the name `MyCustomUmbracoProject`, or which
...
+
+    ```none
+    dotnet new --install Umbraco.Templates::9.0.1 --nuget-source https://api.nuget.org/v3/index.json
+    ```
```
Comment (marcemarc) [inline UmbracoNetCoreUpdates.md:149]:
```suggestion
    ```
Where 9.0.1 is replaced with the latest version that is available that you are trying to install.
```

**Review by marcemarc — CHANGES_REQUESTED** [review]
@JeffreyPerplex thanks for sprinkling some of Seb's secret magic into the docs, we just had some suggestions to the wording, which we have specifically specified in this review.

**Comment by marcemarc** [comment]
@JeffreyPerplex I has tweaked the text - but merged this in! thanking you!

**Pull request merged**